doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -21,65 +21,27 @@ A known issue can list one or both of the following entries:
   Some known issues have a workaround.
   Sometimes, they are discovered later and added over time.
 
-.. raw:: html
-   :file: ../includes/filter.js
-
-.. raw:: html
-
-   Filter by versions:
-
-   <select name="versions" id="versions-select">
-     <option value="all">All versions</option>
-     <option value="v2-4-2" selected>v2.4.2</option>
-     <option value="v2-4-1">v2.4.1</option>
-     <option value="v2-4-0">v2.4.0</option>
-     <option value="v2-3-0">v2.3.0</option>
-     <option value="v2-2-0">v2.2.0</option>
-     <option value="v2-1-4">v2.1.4</option>
-     <option value="v2-1-3">v2.1.3</option>
-     <option value="v2-1-2">v2.1.2</option>
-     <option value="v2-1-1">v2.1.1</option>
-     <option value="v2-1-0">v2.1.0</option>
-     <option value="v2-0-2">v2.0.2</option>
-     <option value="v2-0-1">v2.0.1</option>
-     <option value="v2-0-0">v2.0.0</option>
-     <option value="v1-9-2">v1.9.2</option>
-     <option value="v1-9-1">v1.9.1</option>
-     <option value="v1-9-0">v1.9.0</option>
-     <option value="v1-8-0">v1.8.0</option>
-     <option value="v1-7-1">v1.7.1</option>
-     <option value="v1-7-0">v1.7.0</option>
-     <option value="v1-6-1">v1.6.1</option>
-     <option value="v1-6-0">v1.6.0</option>
-     <option value="v1-5-2">v1.5.2</option>
-     <option value="v1-5-1">v1.5.1</option>
-     <option value="v1-5-0">v1.5.0</option>
-     <option value="v1-4-2">v1.4.2</option>
-     <option value="v1-4-1">v1.4.1</option>
-     <option value="v1-4-0">v1.4.0</option>
-     <option value="v1-3-2">v1.3.2</option>
-     <option value="v1-3-1">v1.3.1</option>
-     <option value="v1-3-0">v1.3.0</option>
-     <option value="v1-2-1">v1.2.1</option>
-     <option value="v1-2-0">v1.2.0</option>
-     <option value="v1-1-0">v1.1.0</option>
-     <option value="v1-0-0">v1.0.0</option>
-     <option value="v0-4-0">v0.4.0</option>
-     <option value="v0-3-0">v0.3.0</option>
-   </select>
-
-..
+.. version-filter::
+  :default: v2-4-2
+  :container: dl/dt
+  :tags: [("wontfix", "Won't fix")]
+
+.. page-filter::
+  :name: issues
+
+  wontfix    Won't fix
+  !wontfix   Fixable
```

Comment:
@wbober, could you confirm if this wording is good for the filter text? Do you have any other suggestions?

Reply:
Do we need `Fixable`? AFAIK, all known issues are fixable until tagged with `Won't Fix`, so why do we need a tag for `!wontfix`? It will only confuse the readers, who will now face issues that are fixable, issues that won't be fixed, and issues that have neither of these tags... Not to mention that `Fixable` could change to `Won't Fix` and we would have to explain why this change happened.

Reply:
@greg-fer There is no visible "Fixable" tag, this is only for the dropdown showing the filter options

Reply:
> Do we need `Fixable`? AFAIK, all known issues are fixable until tagged with `Won't Fix`, so why do we need a tag for `!wontfix`? It will only confuse the readers, who will now face issues that are fixable, issues that won't be fixed, and issues that have neither of these tags... Not to mention that `Fixable` could change to `Won't Fix` and we would have to explain why this change happened.

I'm with @greg-fer on that. I'd just settle on "All" and "Won't fix" filters.

Reply:
"Fixable" has now been removed as a filter option, and the all wontfix-tags has been removed (a screenshot of how it looks can be found in the PR description instead)